backstage/op.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/env bash
+
+# Google
+AUTH_GOOGLE_CLIENT_ID=$(op --account broadinstitute.1password.com read "op://BITS - Backstage/OAuth 2.0 Client ID - Dev/client_id")
+export AUTH_GOOGLE_CLIENT_ID
+AUTH_GOOGLE_CLIENT_SECRET=$(op --account broadinstitute.1password.com read "op://BITS - Backstage/OAuth 2.0 Client ID - Dev/client_secret")
+export AUTH_GOOGLE_CLIENT_SECRET
+# GitHub
+AUTH_GITHUB_CLIENT_ID=$(op --account broadinstitute.1password.com read "op://BITS - Backstage/auth-github-client-id-dev/credential")
+export AUTH_GITHUB_CLIENT_ID
+AUTH_GITHUB_CLIENT_SECRET=$(op --account broadinstitute.1password.com read "op://BITS - Backstage/auth-github-client-secret-dev/credential")
+export AUTH_GITHUB_CLIENT_SECRET
+op --account broadinstitute.1password.com read --out-file ./github-app-backstage-bits-dev-credentials.yaml "op://BITS - Backstage/github-app-backstage-bits-dev-credentials/github-app-backstage-bits-dev-credentials.yaml"
+
+# vim: set et fenc=utf-8 ff=unix ft=sh sts=2 sw=2 ts=2 :
```

Comment:
could we cut this? I don't think this is the appropriate way to enforce style and I'm the only one who uses vim here anyways

Reply:
:shrug: sure, I have that in for when I use vim, but i'll pull it out!